holoviews/tests/plotting/plotly/test_distributionplot.py

```diff
@@ -1,10 +1,19 @@
+from unittest import SkipTest
+
 from holoviews.element import Distribution
 
 from .test_plot import TestPlotlyPlot
 
 
 class TestDistributionPlot(TestPlotlyPlot):
 
+    def setUp(self):
```

Comment:
Instead shouldn't we add `scipy` to the `tests_core` dependencies? Seems like it's pretty important to test the Distribution elements as part of the unit test suite.

Reply:
We use `test` and not `test-core` for our test suite, so we test the `scipy` in our CI. 

In the debug mode, `test_core` was used and gave an error. I think it is acceptable not to have `scipy` in `test-core`.

Reply:
Ok thanks I agree.